man/systemd.time.xml

```diff
@@ -83,13 +82,14 @@
       <listitem><para>days, day, d</para></listitem>
       <listitem><para>weeks, week, w</para></listitem>
       <listitem><para>months, month, M (defined as 30.44 days)</para></listitem>
-      <listitem><para>years, year, y (define as 365.25 days)</para></listitem>
+      <listitem><para>years, year, y (defined as 365.25 days)</para></listitem>
     </itemizedlist>
 
-    <para>If no time unit is specified, generally seconds are assumed,
-    but some exceptions exist and are marked as such. In a few cases
-    <literal>ns</literal>, <literal>nsec</literal> is accepted too,
-    where the granularity of the time span allows for this.</para>
+    <para>If no time unit is specified, generally seconds are assumed, but some exceptions exist and are marked as
+    such. In a few cases <literal>ns</literal>, <literal>nsec</literal> is accepted too, where the granularity of the
+    time span permits this. Parsing is generally locale-independent, non-english units are not accepted.</para>
+
+    <para>Note that time values and their units may appear in any order.</para>
```

Comment:
i think we can drop the entire sentence. I mean, it's not good style saying "3min 5h" anyway, so maybe we better should just handle this silently instead of confusing people with that and pushing them towards actually writing this backwards by documenting it.